wremnants/syst_tools.py

```diff
@@ -4,6 +4,7 @@
 from utilities import boostHistHelpers as hh, common, logging
 from wremnants import theory_tools
 from wremnants.datasets.datagroups import Datagroups
+from wremnants.helicity_utils import *
```

Comment:
I'm against using import * because it becomes unclear what is imported, can we specify the functions that are imported or use ```from remnants import helicity_utils``` and then ```helicity_utils.xyz()```